kernel/src/cmdline/mod.rs

```diff
@@ -29,6 +34,8 @@ impl fmt::Display for Error {
             f,
             "{}",
             match *self {
+                Error::CommandLineCopy => "Failed to copy the command line string to guest memory",
+                Error::CommandLineOverflow => "Command line string overflows guest memory",
                 Error::InvalidAscii => "string contains non-printable ASCII character",
```

Comment:
nit: Let s use same format for all enum values:
```suggestion
                Error::InvalidAscii => "Command line string contains non-printable ASCII character",
```